packages/legacy-compat/src/builders/find-all.ts

```diff
@@ -0,0 +1,53 @@
+import { assert } from '@ember/debug';
+
+import type { StoreRequestInput } from '@ember-data/store';
+import type { FindAllOptions } from '@ember-data/store/-types/q/store';
+import type { TypeFromInstance } from '@warp-drive/core-types/record';
+import { SkipCache } from '@warp-drive/core-types/request';
+
+import { normalizeModelName } from './utils';
+
+export type FindAllRequestInput = StoreRequestInput & {
+  op: 'findAll';
+  data: {
+    type: string;
+    options: FindAllBuilderOptions;
+  };
+};
+
+export type FindAllBuilderOptions = FindAllOptions;
+
+/**
+  This function builds a request config for the given type.
+  When passed to `store.request`, this config will result in the same behavior as a `store.findAll` request.
+  Additionally, it takes the same options as `store.findAll`.
```

Comment:
We should be clear with these that they are basically deprecated from the beginning. They are only here to enable a syntax upgrade on the path to a full upgrade to RequestManager.

Reply:
Do we want to add actual deprecations or just note it in the docs?

Reply:
probably just note in the docs for now